docs/README.md

```diff
@@ -354,25 +354,40 @@ When using a yaml configuration file, which is already available in application
 
 Name | Description | Value | Default |
 --- | --- | --- | --- |
+sip.core.translation | Enable SIP translation | boolean | true
 sip.core.translation.fileLocations | Sets locations of translation bundles | List | classpath:translations/translated-messages, classpath:translations/sip-core-messages |
 sip.core.translation.default-encoding | Sets default encoding | String | UTF-8 |
 sip.core.translation.fallback-to-system-locale | Use system language if none defined | boolean | false |
 sip.core.translation.use-code-as-default-message | If key is not assigned use it in message | boolean | true |
 sip.core.translation.lang | Set language of log messages | String | en |
-sip.core.metrics.external-endpoint-health-check.enabled | Enable health status calculation | boolean | true |
-sip.core.metrics.external-endpoint-health-check.scheduler.fixed-delay | Sets health check execution interval | number | 900000 |
-sip.core.metrics.external-endpoint-health-check.scheduler.initial-delay | Sets health check execution initial delay | number | 5000 |
-sip.core.tracing.enabled | Enable SIP tracing and trace history | boolean | true |
-sip.core.tracing.limit | Sets storage limit in trace history | number | 100 |
+sip.core.tracing.enabled | Enable SIP tracing and trace history | boolean | false |
+sip.core.tracing.exchange-formatter.{property-name} | Sets value for specific property in [ExchangeFormatter](https://www.javadoc.io/static/org.apache.camel/camel-support/3.0.0/org/apache/camel/support/processor/DefaultExchangeFormatter.html) | / | / |
 sip.core.tracing.trace-type | Sets how tracer should behave | String | "*" |
-sip.core.tracing.exchange-formatter.{property-name} | Sets value for specific property in ExchangeFormatter | / | / |
-sip.testkit.enabled | Enables SIP Test Kit | boolean | true |
+sip.core.actuator.enabled | Enable Spring actuator | boolean | true |
+sip.core.proxy.enabled | Enable SIP proxy | boolean | true |
```

Comment:
Please remove this line.
Proxy is internal architecture concept for now, we don't mention it in public docs.

Reply:
Removed.